docs/guides/index.md

```diff
@@ -10,10 +10,11 @@ Here you will find a set of guides to help you with common (and not so common) t
 
 | Name                                                                                                                                     | Description                                                                                    |
 | ---------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| [Configure Site-to-Site Connectivity for APIs](/guides/site-to-site-apis)                                                                | Learn how to access APIs at external sites with mTLS encryption                                |
+| [Configure Site-to-Site Connectivity for Databases](/guides/site-to-site-dbs)                                                            | Learn how to access databases at external sites with mTLS encryption                           |
```

Comment:
bumped these two new docs up to the first of the list at the request of @shubcodes / shubernetes (same thing)